src/ck-core/ckcheckpoint.C

```diff
@@ -227,7 +227,7 @@ void CkCheckpointMgr::Checkpoint(const char *dirname, CkCallback cb, bool _reque
     if (pending_realloc_state == REALLOC_IN_PROGRESS) {
       // After restarting from this AtSync checkpoint, resume execution along the
       // normal path (i.e. whatever the user defined as ResumeFromSync.)
-      CkCallback resumeFromSyncCB(CkIndex_LBDatabase::ResumeClients(), _lbdb);
+      CkCallback resumeFromSyncCB(CkIndex_LBManager::ResumeClients(), _lbdb);
```

Comment:
This is now called on LBManager not _lbdb (which is either named inaccurately, or a proxy for LBDatabase?)